hal/src/samd51/sercom/pads.rs

```diff
@@ -279,32 +284,38 @@ pad!(Sercom6Pad3 {
 // sercom7[2]:  PB18:D   PC10:D   PC14:C   PD10:C
 // sercom7[3]:  PC11:D   PC15:C   PD11:C   PB19:D
 
-#[cfg(feature = "samd51p19a")]
+#[cfg(any(feature = "samd51p19a", feature = "samd51n20a"))]
 pad!(Sercom7Pad0 {
     Pb21(PfD),
     Pc12(PfC),
+    #[cfg(feature = "samd51p19a")]
     Pd8(PfC),
+    Pb30(PfC),
```

Comment:
If I understand the datasheet correctly, PB30 and PB31 are valid pins for SERCOM7, but they are missing here. Was that intentional or a mistake? I added them as a separate commit, so I can easily split the two commits into two PRs if required.

Reply:
You appear to be correct, I would imagine this was a mistake.